### ISSUE TITLE
train and test data for skeletongait++_CCPG

### ISSUE THREAD

**Issue by nyanmn** [ShiqiYu/OpenGait#233]
## System information (version)

- Pytorch => 2.3.0
- Operating System / Platform => Ubuntu 22.04
- Cuda => 12.2

## Detailed description
Dataset split between train and test data.
I am using CCPG dataset for skeletongait++.
skeletongait++_CCPG has CCPG.json for train and test set. 000-099 are used for training and 100-199 are for test.
CCPG has 200 different people.
 skeletongait++ is gait recognition algorithm. So how model can recognize gait for a particular person using different people for training and testing.

## Issue submission checklist
 - [ ] I checked the problem with documentation, FAQ, issues, and have not found solution.

**Comment by nyanmn** [comment]
My understanding for gait recognition is that train the model with person A and test the model with new images of that same person A to see whether the trained model can recognize that test person has similar gait as trained person A.
